app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/actions/SubscriptionDetailsAction.scala

```diff
@@ -32,35 +35,37 @@ import uk.gov.hmrc.play.HeaderCarrierConverter
 
 import scala.concurrent.{ExecutionContext, Future}
 
-final case class RequestWithSubscriptionDetails[A](
+final case class RequestWithSubscriptionData[A](
     subscriptionDetails: SubscriptionDetails,
```

Comment:
Should SubscriptionDetails be renamed to SubscriptionData inline with this change?

Reply:
This was something I changed while experimenting with something but forgot to change back - I meant for this to remain the same as `RequestWithSubscriptionDetails`